mysql/time.go

```diff
@@ -1375,3 +1375,40 @@ func ExtractTimeValue(unit string, format string) (int64, int64, int64, time.Dur
 		return 0, 0, 0, 0, errors.Errorf("invalid singel timeunit - %s", unit)
 	}
 }
+
+// IsHourMinuteSecondUnit returns true when unit is interval unit with hour, minutes or seconds.
+func IsHourMinuteSecondUnit(unit string) bool {
```

Comment:
For go time package, it uses Date() for year, month, and day, uses Clock() for hour, minute and second, so I think here we can use IsClockUnit instead.